talking-about-their-pronouns.md

```diff
@@ -0,0 +1,38 @@
+# Asking someone's pronouns
+
+## Over Slack / in conversation
+
+What: Sorry, just to check so I can get them right, do you mind me asking what your pronouns are?
```

Comment:
```suggestion
When: In introducing oneself to someone new
What: Hi, I'm Janet Jackson, and I go by `they/them` pronouns. How should I refer to you?

When: For someone you already know
What: By the way, I use `he/him` pronouns. How should I refer to you?
```
Adapted from https://pronouns.org/asking. You may be asking someone who doesn't know about pronouns, so by offering an example of the expected answer, it might help the prompt.

I also have a personal bugbear about saying "sorry" when I've not done anything wrong! :sweat_smile: